Add the test project to Input_Test folder

JiaUnity opened the pull request:
Just add the project into the latest Develop.

JiaUnity committed:
Add the test project to Input_Test folder

Rene-Damm commented on Assets/Input_Test/Scripts/02_Input/ClickOnDropdown.cs:
```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class ClickOnDropdown : MonoBehaviour {
```
Please run format.sh or format.ps1 found in the root of the repo. Otherwise the next commit running the format tools will have to recommit a reformat of all the files here.

Rene-Damm commented on Assets/Input_Test/Scripts/02_Input/Gamepad4InputManager.cs:
```diff
@@ -0,0 +1,99 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Gamepad4InputManager : MonoBehaviour
```
Naming seems somewhat confusing as there's also an InputManager class in the scripts. `GamepadForOldInputSystem` and `GamepadForNewInputSystem` seem more obvious.

JiaUnity commented:
Let me clean the project a little bit more first.

Rene-Damm commented on Assets/Input_Test/Created by @jia.txt:
```diff
@@ -0,0 +1 @@
+Please contact jia@unity3d.com for any concern/comment or poke me on Slack.
```
Some quick instructions and what-is-this would be great I think. So far, the QA material is mostly an assortment of what appears like random scenes to the user.

Rene-Damm reviewed (commented):
Please move entire Input_Test folder into Assets/QA. Also, some indication as to why/how this is different from Assets/QA/Tests would be great.

JiaUnity committed:
- Separate scenes, since UI doesn't work with Input System yet.
- Disable old Input Manager scripts, etc.
- Attach mouse & controller stick movement visualization to Input System (was controlled by Input Manager)

*NOTE: Not all UI related to Input Manager is removed. Just ignore them during test.

JiaUnity committed:
- Move project to the right folder
- Better introduction of the project
- Formatting
